modules/home-manager/kitty.nix

```diff
@@ -7,5 +7,5 @@ in
 {
   options.programs.kitty.catppuccin = ctp.mkCatppuccinOpt { name = "kitty"; };
 
-  config.programs.kitty = lib.mkIf enable { theme = "Catppuccin-${ctp.mkUpper cfg.flavor}"; };
+  config.programs.kitty = lib.mkIf enable { themeFile = "Catppuccin-${ctp.mkUpper cfg.flavor}"; };
```

Comment:
This should be a path, I'd assume? You are still just passing the name of a theme.